frontend/vue/mixins/menu.ts

```diff
@@ -1,62 +0,0 @@
-import { Vue } from 'vue-class-component'
-
-import {
-  ORDERED_COMMUNITY_SUB_LINKS,
-  HOME_LINK,
-  COMMUNITY_LINK,
-  TUTORIALS_LINK,
-  DOCUMENTATION_LINK,
-  LEARN_LINK,
-  OVERVIEW_LINK,
-  NavLink
-} from '../../constants/menuLinks'
```

Comment:
I believe that most of the contents of the `frontend/constants/menuLinks.ts` file can now be removed.

Only `NavLink` seems to still be used.

Reply:
thanks @eddybrando ! resolved by https://github.com/Qiskit/platypus/pull/1555/commits/a5b40af0893161f8c61c80ec198144706f8c390d